packages/nextjs/pages/index.tsx

```diff
@@ -322,6 +322,40 @@ const Home: NextPage<{
         </div>
       </div>
 
+      {/* Nodes section */}
+      <div className="bg-[#EBEBEB]">
+        <div className="mx-auto lg:max-w-7xl container flex flex-col lg:flex-row px-10 py-14 lg:py-16 xl:pl-24 lg:px-14 text-black">
+          {/* First column */}
+          <div className="flex flex-col flex-wrap">
+            <div className="flex items-start p-3 w-full lg:border-t lg:border-l lg:border-r-0 lg:border-b-0 border border-black">
+              <Image src="/assets/client-logo.svg" alt="client logo" width={250} height={250} />
+            </div>
+            <div className="flex flex-row lg:border-y lg:border-l lg:border-r-0 border-x border-black">
```

Comment:
Adding `flex-grow` to this div will make the inner pink and green boxes "expand" to fill the rest of the space

Reply:
Also I noticed on mobile, the inner boxes are side-by-side which makes it look strange. If you want them stacked, you can change the `flex` to `md:flex` so that they only go side-by-side on larger screens.

![Screenshot 2024-12-03 at 7 03 25 AM](https://github.com/user-attachments/assets/e31f536d-fe6f-43ea-b199-5ec24b11d2bd)